technische-universitat-munchen-controlling.csl

```diff
@@ -7,7 +7,7 @@
     <id>http://www.zotero.org/styles/technische-universitat-munchen-controlling</id>
     <link href="http://www.zotero.org/styles/technische-universitat-munchen-controlling" rel="self"/>
     <link href="http://www.zotero.org/styles/universitat-freiburg-geschichte" rel="template"/>
-    <link href="http://www.zotero.org/styles/hwr-berlin" rel="template"/>
+    <link href="http://www.zotero.org/styles/hochschule-fur-wirtschaft-und-recht-berlin" rel="template"/>
```

Comment:
We definitely do, yes -- without actually having looked at the code, I'm guessing the test might just look if the first template exists. This style has two.